devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.fast-jar

```diff
@@ -21,10 +21,10 @@
 # docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/{project.artifact-id}-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM {dockerfile.fast-jar.from}
```

Comment:
fyi, this is not the image we need to paramaterize currently. its nice its there but this isn't related to native builds.

Reply:
ah ok, well :)